systers_portal/static/css/style.css

```diff
@@ -234,6 +234,22 @@ table.decoration-none  tr a {
   font-size: 1.3em;
 }
 
+/* Meetup Location CSS
+------------------------------------------------- */
+.box-boundary{
+  outline: 1px solid black;
```

Comment:
Following the conventions in the CSS file, could you please add a space before "{". Thank you.